integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java

```diff
@@ -335,6 +335,7 @@ public interface TestConstants {
       "-Dweblogic.security.SSL.ignoreHostnameVerification=true -Dweblogic.security.TrustKeyStore=DemoTrust";
 
   public static final boolean WEBLOGIC_SLIM = WEBLOGIC_IMAGE_TAG.contains("slim");
+  public static final boolean WEBLOGIC_12213 = WEBLOGIC_IMAGE_TAG.contains("12.2.1.3");
```

Comment:
Add some comment why we are skipping  as follows  ..
"application deployment through REST interface issue and is included in WebLogic 12.2.1.3.0 CPU's"

Reply:
added comments in the extensions class as that's where we disable